fastestimator/estimator.py

```diff
@@ -39,10 +38,11 @@
 from fastestimator.trace.io.traceability import Traceability
 from fastestimator.trace.trace import EvalEssential, Logger, PerDSTrace, TestEssential, Trace, TrainEssential, \
     sort_traces
+from fastestimator.util.base_util import NonContext, Suppressor, to_list, to_set
 from fastestimator.util.data import Data, FilteredData
 from fastestimator.util.traceability_util import traceable
 from fastestimator.util.util import draw
-from fastestimator.util.base_util import to_set, to_list, NonContext, Suppressor
+from tensorflow.python.distribute.input_lib import DistributedDataset
```

Comment:
this import sorting isn't right, external modules should appear before fe imports